boring-sys/build.rs

```diff
@@ -464,13 +465,14 @@ fn main() {
         "x509v3.h",
     ];
     for header in &headers {
-        builder = builder.header(
-            Path::new(&include_path)
-                .join("openssl")
-                .join(header)
-                .to_str()
-                .unwrap(),
-        );
+        let path = Path::new(&include_path).join("openssl").join(header);
+        let path_str = path.to_str().unwrap();
+        if !path.exists() {
+            eprintln!("included header does not exist: {:?}. Skipping.", path_str);
+            continue;
+        }
+        eprintln!("including bindings for header: {:?}.", path_str);
+        builder = builder.header(path_str);
```

Comment:
What's the reason for this change? All those files must exist, so a downgrade to a warning looks a bit suspicious.